packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2599,10 +2608,12 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             deserializedContent,
             metadata,
             localOpMetadata,
+            referenceSequenceNumber: this.deltaManager.lastSequenceNumber,
         });
         if (this._flushMode !== FlushMode.TurnBased) {
             this.flush();
         } else if (!this.flushTrigger) {
+            this.flushTrigger = true;
```

Comment:
This looks like new code. Why is this needed?

Reply:
I missed it in my previous check in. Without that we would have a lot of promises (one per on) that do empty flushes (as only one of them does work).